From: George McCollister <george.mccollister@gmail.com>
Date: Wed, 27 Oct 2010 23:17:40 +0200
Subject: [PATCH] include stdio.h

fcgi failed to compile. EOF wasn't defined because stdio.h wasn't included.

Signed-off-by: George McCollister <george.mccollister@gmail.com>
---
 include/fcgiapp.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/include/fcgiapp.h b/include/fcgiapp.h
index d7236f6..c39a671 100644
--- a/include/fcgiapp.h
+++ b/include/fcgiapp.h
@@ -34,6 +34,8 @@
 extern "C" {
 #endif
 
+#include <stdio.h>
+
 /*
  * Error codes.  Assigned to avoid conflict with EOF and errno(2).
  */
-- 
1.7.2.3

